Formula/lammps.rb

```diff
@@ -1,23 +1,26 @@
 class Lammps < Formula
   desc "Molecular Dynamics Simulator"
   homepage "https://lammps.sandia.gov/"
-  url "https://github.com/lammps/lammps/archive/stable_29Sep2021.tar.gz"
+  url "https://github.com/lammps/lammps/archive/stable_29Sep2021_update1.tar.gz"
   # lammps releases are named after their release date. We transform it to
   # YYYY-MM-DD (year-month-day) so that we get a sane version numbering.
   # We only track stable releases as announced on the LAMMPS homepage.
   version "2021-09-29"
-  sha256 "2dff656cb21fd9a6d46c818741c99d400cfb1b12102604844663b655fb2f893d"
+  sha256 "5000b422c9c245b92df63507de5aa2ea4af345ea1f00180167aaa084b711c27c"
   license "GPL-2.0-only"
-  revision 1
+  revision 2
```

Comment:
`revision` is for Homebrew-specific versioning. Upstream have a detectable version change, so let's use `-update1` in the version. If we can drop the `-`s in the date part, that might be useful too, since that seems to be the version scheme most other distros use: https://repology.org/project/lammps/versions

Reply:
Makes sense to me. I updated the `version`, removed the `revision` (since it's a version bump), and modified the `livecheck` block to capture the optional `update` part (returning `20210929-update1` for the latest `stable_29Sep2021_update1` tag).